core/pnet/codec.go

```diff
@@ -31,7 +31,7 @@ func expectHeader(r *bufio.Reader, expected []byte) error {
 		return err
 	}
 	if !bytes.Equal(header, expected) {
-		return fmt.Errorf("expected file header %s, got: %s", pathPSKv1, header)
+		return fmt.Errorf("expected file header %s, got: %s", expected, header)
```

Comment:
Makes sense to change this to expected but I am not the expert here, cc @marten-seemann